agent/src/util/config_action.rs

```diff
@@ -154,6 +158,16 @@ async fn handle_config(
                 "handle_config - modified Configuration {:?}",
                 config.metadata.name,
             );
+            let do_recreate = should_recreate_config(&config, config_map.clone())
+                .await
+                .unwrap();
+            if !do_recreate {
+                trace!(
+                    "handle_config - config {:?} has not changed. ignoring config modified event.",
+                    config.metadata.name,
+                );
+                return Ok(());
+            }
```

Comment:
Just realized, I should probably move this above the info log and maybe change my trace log to an info log?

Reply:
Actually, I'll keep my log message a trace, but I think it should be before the info log. The only reason I was considering it an info was because i thought the "something happened to a configuration" log was an info, and I didn't want a user to be left with a message saying that something happened, but then potentially no log saying what happened. The gist is, this message about ignoring the event should be at the same log level (or more verbose) as the "something happened to a configuration" log message.

Reply:
That methodology makes sense. We need to do an overhaul of our logging at some point so I understand if the current strategy seems inconsistent. What you have right now is great!